RoboFile.php

```diff
@@ -553,6 +556,16 @@ public function releaseCheckIssues($version = null) {
     }
   }
 
+  public function releasePrepareGit() {
+    $git_status = $this->taskGitStack()
+      ->printOutput(false)
+      ->exec('git status --porcelain')
+      ->run();
+    if (strlen(trim($git_status->getMessage())) > 0) {
+      throw new \Exception('Please make sure your working directory is clean before running release.');
```

Comment:
In some other places we use `yell()` and `exit(1)` but I guess this will have the same result?